openjdk.test.load/src/test.load/net/adoptopenjdk/stf/LambdaLoadTest.java

```diff
@@ -49,7 +49,7 @@ public void execute(StfCoreExtension test) throws StfException {
 				.setSuiteRandomSelection();		  			// Randomly pick the next test each time
 		
 		test.doRunForegroundProcess("Run lambda and stream load test", "LT", Echo.ECHO_ON,
-				ExpectedOutcome.cleanRun().within("30m"), 
+				ExpectedOutcome.cleanRun().within("60m"), 
```

Comment:
Trying to distinguish at this test level if we are running on -xint can be an overkill. Increasing the timeout doesn't mean the test will hang around for this long though, this is only going to take effect when the test takes long (e.g. in case of -xint).

Reply:
Sure, let's give it try. Thanks.